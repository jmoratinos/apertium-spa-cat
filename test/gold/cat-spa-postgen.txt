[FE6jPnAZmlXf]
pero [/option]
[/FE6jPnAZmlXf]
