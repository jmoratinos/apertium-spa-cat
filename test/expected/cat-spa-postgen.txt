[FE6jPnAZmlXf#0] pero
[/FE6jPnAZmlXf]
